uber/models.py

```diff
@@ -133,20 +128,22 @@ def process_bind_param(self, value, dialect):
         return value if isinstance(value, str) else ','.join(value)
 
 
-@declarative_base
+default_metadata = MetaData(
+    naming_convention=immutabledict({
+        'unnamed_ck': check_constraint_naming_convention,
+        'ix': 'ix_%(column_0_label)s',
+        'uq': 'uq_%(table_name)s_%(column_0_name)s',
+        'ck': 'ck_%(table_name)s_%(unnamed_ck)s',
+        'fk': 'fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s',
+        'pk': 'pk_%(table_name)s'}))
```

Comment:
Why these overrides?  What's the difference between this and the default behavior and what we have here, and why do we care about that difference?  (A one-line comment explaining that in the code might be helpful, regardless of the answer.)